x-pack/plugins/siem/public/components/stat_items/index.test.tsx

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { mount, ReactWrapper } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import * as React from 'react';
+
+import { StatItemsComponent, StatItemsProps } from '.';
+import { BarChart } from './barchart';
+import { AreaChart } from './areachart';
+import { EuiHorizontalRule } from '@elastic/eui';
+// @ts-ignore
+import { EuiAreaSeries, EuiBarSeries } from '@elastic/eui/lib/experimental';
```

Comment:
Are these unused imports necessary? I was able to remove them and these tests passed.